pootle/apps/pootle_project/views.py

```diff
@@ -171,6 +171,10 @@ def dispatch(self, request, *args, **kwargs):
     def pootle_path(self):
         return self.object.pootle_path
 
+    @property
+    def request_path(self):
+        return "/projects/%(project_code)s/%(dir_path)s" % self.kwargs
```

Comment:
This is `ProjectTranslateView` so I don't think it is going to retrieve `get_stats` in any case.